content/200-orm/200-prisma-client/100-queries/037-relation-queries.mdx

```diff
@@ -24,6 +24,65 @@ Nested reads allow you to read related data from multiple tables in your databas
 - Use [`include`](/orm/reference/prisma-client-reference#include) to include related records, such as a user's posts or profile, in the query response.
 - Use a nested [`select`](/orm/reference/prisma-client-reference#select) to include specific fields from a related record. You can also nest `select` inside an `include`.
 
+### Relation load strategies (Preview)
+
+Since [5.9.0](https://github.com/prisma/prisma/releases/tag/5.9.0), you can decide on a per-query-level _how_ you want Prisma Client to execute a relation query (i.e. what _load strategy_ should be applied) via the `relationLoadStrategy` option.
+
+Because the `relationLoadStrategy` option is currently in Preview, you need to enable it via the `relationJoins` preview feature flag in your Prisma schema file:
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["relationJoins"]
+}
+```
+
+After adding this flag, you need to run `prisma generate` again to re-generate Prisma Client. Also note that this feature is currently only available on PostgreSQL and CockroachDB, support for other DBs is coming soon.
+
+Prisma Client supports two load strategies for relations:
+
+- `join` (default): Uses a database-level `LATERAL JOIN` and fetches all data with a single query to the database.
+- `query`: Sends multiple queries to the database (one per table) and joins them on the application level.
+
+Another important difference between these two options is that the `join` strategy uses JSON aggregation on the database level. That means that it creates the JSON structures returned by Prisma Client already in the database which saves computation resources on the application level.
+
+> **Note**: Once `relationLoadStrategy` moves from [Preview](/orm/more/releases/preview) into [General Availability](/orm/more/releases/#generally-available-ga), the default will switch to `join`.
```

Comment:
The default is already `join` for the feature - I think you want to say that it will universally become the default load strategy?

Reply:
Ah, yes. I think you _can_ already read the current sentence like that but also see how it's a bit ambiguous. Is this clearer?

```suggestion
> **Note**: Once `relationLoadStrategy` moves from [Preview](/orm/more/releases/preview) into [General Availability](/orm/more/releases/#generally-available-ga), `join` will universally become the default for all relation queries.
```

Reply:
Yes, much better 👍